adagucserverEC/Definitions.h

```diff
@@ -188,3 +188,10 @@
 
 // #define ADAGUC_USE_GDAL
 #endif
+
+// HTTP status codes
+// Success
+#define SUCCESS_200_OK 0
```

Comment:
Would it be an idea to prefix these three with  HTTP_STATUSCODE_<...> ?

Reply:
Sure, will change that!